app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb

```diff
@@ -32,6 +32,12 @@
               title: "Design and technology",
               description: "As a design and technology teacher, you'll help pupils to turn their ideas into reality.",
               path: "/life-as-a-teacher/explore-subjects/design-and-technology",
+            )) %>
+              <%= render Categories::CardComponent.new(heading_tag: "h2", card:
```

Comment:
If this is the next subject PR to go out, could we reorder Languages and History on the subjects page?